integration-libs/opf/core/events/opf-ui-events.listener.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, OnDestroy } from '@angular/core';
+import { EventService } from '@spartacus/core';
+import { OpfUiClearEvent } from '@spartacus/opf/root';
+import { Subscription } from 'rxjs';
+import { OpfService } from '../facade/opf.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OpfUiEventListener implements OnDestroy {
+  protected subscriptions = new Subscription();
+
+  constructor(
+    protected eventService: EventService,
+    protected opfService: OpfService
+  ) {
+    this.onOpfUiClear();
+  }
+
+  /**
+   * Registers events for the cart base actions.
```

Comment:
I think this comment does not refer to functionality.

Reply:
changed, thanks